exec/csnex/Bytecode.cs

```diff
@@ -1,12 +1,13 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Diagnostics;
+using System.Reflection;
 
 namespace csnex
 {
     public class Bytecode
     {
-        private const int BYTECODE_VERSION = 3;
+        public static int BYTECODE_VERSION = Assembly.GetExecutingAssembly().GetName().Version.Revision;
```

Comment:
I thought you were kidding when you said you were going to do this. Nope, make the bytecode version a literal constant like it was.

Reply:
I don't disagree.  I thought I'd try it this way, so that if there was ever any question, you could easily look at the csnex version and know what version of the bytecode it can execute.  Too bad I can't do the opposite here either, and use the `BYTECODE_VERSION` in the `AssemblyVersion`